### DISCUSSION TITLE
Android app: Is PRO-license per unit or per user?

### DISCUSSION THREAD

**Discussion by nykjell** [fhdm-dev/scoppy#142] [General]
Can i use the app fully on two Android devices (phone and tablet), or do I need to buy separate license for each device?
   (I did not find any information or discussion on this.)

Best regards,
 /Kjell

**Comment by kd7eir** [comment]
You pay for the license through the Google Play Store, so you can use it on
any device that is signed in with the same Google account that you
purchased it with.

**Reply by nykjell** [reply]
Great, thanks! 
I have only tested as an oscilloscope with a Raspberry PI pico as "frontend".
But now I need two channels and other "pro" functions.
Best regards,
 /Kjell

**Comment by nykjell** [comment]
Of course it works that way with purchases through Google play.
Now also confirmed on second unit. :)
I´ll try to close discussion as solved ...
